pkg/web/templates/apikeys.html

```diff
@@ -21,6 +21,7 @@ <h1 class="font-bold text-2xl">API Keys</h1>
 
 <!-- Target to display other api key management modals -->
 <dialog id="apikey_modal" class="modal"></dialog>
+<dialog id="revoke_key_modal" class="modal"></dialog>
```

Comment:
Initially, the revoke api key modal was swapped into the dialog with the `apikey_modal` ID. Since the bug didn't always occur during testing, I decided to create a separate `dialog` element to swap the revoke confirmation modal into as a way to hopefully resolve the issue.